main.py

```
@@ -32,80 +98,414 @@ def parse_mr_url(url):
+        elif "/test/" in path or path.endswith("Test.java"):
+            priority = 2
+        elif path.endswith((".yml", ".yaml", ".properties", ".xml")):
+            priority = 3
+        else:
+            priority = 1
+        
+        return (priority, diff_size)
+    
+    return sorted(changes, key=sort_key)
+
+def validate_suggestion_relevance(suggestion_text, review_messages):
```

Ruff (0.15.2)

[warning] Unused function argument: `suggestion_text`

(ARG001)